package.json

```diff
@@ -16,6 +16,7 @@
   "license": "ISC",
   "dependencies": {
     "@defillama/sdk": "latest",
+    "@drift-labs/sdk": "^2.35.1-beta.6",
```

Comment:
I try to aviod adding new dependencies where i can, can the config be pulled from github or some api?